src/models/ability.ts

```diff
@@ -25,6 +26,7 @@ export interface Ability extends Element {
 	distance: AbilityDistance[];
 	target: string; // Creature, Object, Enemy, Ally, Self, All
 	cost: number;
+	signature: boolean;
```

Comment:
Is there any ability with a cost that's a signature ability? Or any ability without a cost that's not a signature ability?

Reply:
Hmm. I'd understood that any no-cost ability was essentially a signature ability, but if not, then cost should be `number | 'signature'`.

Reply:
Not as many as I thought, but they do exist. (A lot of the things I thought were 0-cost non-signature actions were maneuvers.)

e.g. Flesh Mournling's Horrid Wail, Shambling Mound's Seismic Slam, Troll Butcher's Rotten Scraps